process_sarif/build_results_archive.py

```diff
@@ -42,7 +43,15 @@ def main():
         archive.add('build-results-archive')
 
         # TODO(Steven!) Add colcon-build-cmd, colcon-test-cmd, and process-sarif-cmd.
+        
         # TODO(Steven!) Add vcs-export-exact.repos
+        # Assume we are at the workspace root, since log/build_results_archives is relative to ws_root.
+        repos = subprocess.call(["vcs", "export", "--exact", "src"])
```

Comment:
At the very least, it's probably worth making this a `check_call()` so that the entire process fails if the vcs-export does. I had been thinking to use vcs as a [runnable module](https://peps.python.org/pep-0338/) rather than shelling out directly but I'm not sure that it's ultimately that important